src/lib/RouteQueryActionsController.ts

```diff
@@ -38,8 +45,29 @@ export const RouteQueryActionsController = (() => {
     /**
      * Take action after opening the link copied in the transfer receive modal.
      */
-    transferSend: () => {
+    transferSend: (_, query) => {
       const { openModal } = useModals();
+      const { setActiveAccountByProtocol } = useAccounts();
+
+      /**
+       * Determine the active protocol for the current transfer
+       */
```

Comment:
I suggest to use regular inline comment ( `// ...`) for such a tiny lines. DOCBLOC comments in my opinion are reserved for things like functions, methods and object properties.